src/posix/platform/netif.cpp

```diff
@@ -1751,6 +1751,12 @@ static void HandleNetlinkResponse(struct nlmsghdr *msg)
         requestPayloadLength = NLMSG_PAYLOAD(&err->msg, 0);
     }
 
+    if (!(msg->nlmsg_flags & NLM_F_ACK_TLVS))
+    {
+        // there's no tlv attributes, so just dump the error we got
+        goto dowarn;
```

Comment:
nit: to avoid the additional `goto`, can we just move the block below into this if-statement?

Reply:
fixed.